.github/workflows/cra.yaml

```diff
@@ -97,11 +97,32 @@ jobs:
           set -o pipefail
           touch test.log
           make test | tee -a test.log
+      - name: filter out coverage
+        run: |
+          #!/bin/bash
+
+          filters=(
+              "mock"
+              "test"
+          )
+
+          input="./components/compass-runtime-agent/filtered.cov"
```

Comment:
shouldn't this be `cover.out` instead of `filtered.out`?

Reply:
`cover.out` would make more sense, but we are using `filtered.out`.
https://github.com/kyma-project/application-connector-manager/blob/125c014b49a3d37f41f8f0b930f71a3337d1c2be/components/compass-runtime-agent/Makefile#L5-L10

Our coverage file naming is inconsistent across all components, sometimes it's `coverage.txt`, sometimes `cover.out`, and here for some reason `filtered.out`. 
Would be nice to normalise it, but I was afraid of breaking other pipelines before we know if coveralls works.

Reply:
Could we rename it to make it more consistent, as we are already in the context? I fear that it might lead to some time consuming confusions in the future if we have to work in this area again.

Reply:
Ah, I meant to change it, but push failed 🤦

Do you think it's worth it normalizing the coverage file name between `cover.out`, `coverage.txt` and other relatively reasonable options throughout all our services?